### DISCUSSION TITLE
How to add items to the nav menu?

### DISCUSSION THREAD

**Discussion by nguillaumin** [atari-legend/atari-legend#186] [Q&A] [answered]
Hi all, @stgraveyard , I've been working on a magazine section for the site. It's almost ready to review :smile: 

One issue I have is with the nav menu: I need to add a new item "Magazines" (or "Mags" maybe for a shorter version), but the menu is already full and adding a new entry will not fit in the standard screen width. I have tried to shrink the menu a bit to make some room (reduce padding, smaller font). Here's how it looks:

![menu-small](https://user-images.githubusercontent.com/762512/185737010-308c8345-03c5-4a5f-9904-8aef7b01bace.png)

Compared to the regular menu:

![menu-regular](https://user-images.githubusercontent.com/762512/185737015-e9a3f36e-a43a-44ee-9047-a837166cb6de.png)

Full page screenshots:

![full-small](https://user-images.githubusercontent.com/762512/185737084-2634644e-005f-4781-9c58-e5f653119dcf.png)

![full-regular](https://user-images.githubusercontent.com/762512/185737090-b182bb73-19aa-40ea-ae19-d84cb0cb9ad1.png)

I'm not sure if I like it. It feels a bit too small, but maybe it's just that I'm too used to the regular menu?

Another option would be to ditch the "Forum" entry and replace it with "Mags", since Forum is just a link to AF and not a proper menu entry. Or another alternative would be to come up with a completely new menu / header concept...

Let me know what you think?

Nico

**Comment by stgraveyard** [comment, marked as answer]
Nico, when looking at the buttons, it seems to me the only thing that went smaller is the heigt of the buttons? This is clearly visible with the search bar, which now looks ugly. Would it be perhaps possible to keep the font size, but make the distance between the text and the button side borders smaller? 

Another option is to remove the 'Links' button, as I'm sure nobody uses this section and we have the 'HOTLINKS' at the main page. 
The FORUM button I would leave. Atari-Forum is part of the AL family. 

Also, this is how I'm seeing this beautiful site these days : 

![image](https://user-images.githubusercontent.com/4339294/185739242-ccd6c1e2-1259-4b43-8603-b4dc6a8b67d8.png)

Space enough in this version ;-)

**Reply by nguillaumin** [reply]
The font size is smaller, but only by 0.25rem so it's not very visible.

If I only reduce the padding, it gets too busy on smaller screens I think:

![nav-busy](https://user-images.githubusercontent.com/762512/185751771-40a1ae2e-7204-4f19-a86f-24ea8155c1d2.png)

The same resolution but with the smaller font-size and padding:

![nav-reduced](https://user-images.githubusercontent.com/762512/185752096-239454e3-fd1b-410e-9837-ae3fe7e2c92e.png)

I don't like the idea of removing the links menu but keeping all the pages. I'd rather that all sections from the site are accessible from the menu. But perhaps we can completely remove the links section from the site? Looking at the stats since 2020 only 0.3% visitors went to the link section, and we don't really keep it up to date...

**Reply by stgraveyard** [reply]
Nico, can you deploy on dev for me to have a look real time on my new screen and on my laptop. Second shot looks pretty good to me...

**Reply by nguillaumin** [reply]
The second one is the same as in my first post, just shown on a smaller resolution. With a larger screen it's like in my first screenshot, it looks a bit small... I was able to tweak the search box though:

![nav2](https://user-images.githubusercontent.com/762512/185792024-a05c609e-1550-469b-b64e-0ec79f4b79a0.png)

I will post on dev when I it's close to ready, because once it's on dev I cannot easily revert it if we need to make another fix. I'll let you know.

**Reply by nguillaumin** [reply]
New menu with smaller font and smaller padding is now deployed on dev.

**Reply by stgraveyard** [reply]
I guess it is all a matter in which resolution you are watching and on what kind of screen. 

This is how it now looks on my screen : 

![image](https://user-images.githubusercontent.com/4339294/185802578-5908faee-d826-49e0-9f42-d758aa81f918.png)

I can get used to it, it is a bit small. 

When I zoom to 150% to fill screen : 

![image](https://user-images.githubusercontent.com/4339294/185802630-de1ee806-904c-456c-a6c7-7ecdc93e061f.png)

If at all possible, giving the buttons a bit more height, would seem perfect. If not, so be it, I guess...I can live with it...

**Reply by nguillaumin** [reply]
I have added more padding, please check. It's almost the same height as before, except a little bit less because the font is smaller.

**Reply by stgraveyard** [reply]
I personally, most definitely like this one. If it is ok for you it can go to PROD like this once the section is done. Thanks.

**Reply by nguillaumin** [reply]
Great thanks, then I close this thread.

**Comment by stgraveyard** [comment]
BTW, I can not wait to start testing this new section! If we have a good mag section, it might help me with my research making videos as well ...
